commit 1d40f33eba5b400ef23c67be7442b06df22abb60
Author: James Crook <james.k.crook@gmail.com>
Date:   Wed Feb 13 08:46:03 2019 +0000

    Rename 64 bit ffmpeg libs.
    
    Also change the prompting to suggest the new/expected name.

--- src/FFmpeg.cpp
+++ src/FFmpeg.cpp
@@ -641,17 +641,16 @@ bool FFmpegLibs::FindLibs(wxWindow *parent)
    wxString path;
    wxString name;
 
-   // If we're looking for the lib, use the standard name, as the
-   // configured name is not found.
-   name = GetLibAVFormatName();
    wxLogMessage(wxT("Looking for FFmpeg libraries..."));
    if (!mLibAVFormatPath.empty()) {
       wxLogMessage(wxT("mLibAVFormatPath ('%s') is not empty."), mLibAVFormatPath);
       const wxFileName fn{ mLibAVFormatPath };
       path = fn.GetPath();
+      name = fn.GetFullName();
    }
    else {
       path = GetLibAVFormatPath();
+      name = GetLibAVFormatName();
       wxLogMessage(wxT("mLibAVFormatPath is empty, starting with path '%s', name '%s'."),
                   path, name);
    }
--- src/FFmpeg.h
+++ src/FFmpeg.h
@@ -288,22 +288,16 @@ public:
 
    wxString GetLibAVFormatName()
    {
-      if (sizeof(void*) == 8)
-         return (wxT("ffmpeg.") wxT(AV_STRINGIFY(LIBAVFORMAT_VERSION_MAJOR)) wxT(".64bit.dylib"));
       return (wxT("libavformat.") wxT(AV_STRINGIFY(LIBAVFORMAT_VERSION_MAJOR)) wxT(".dylib"));
    }
 
    wxString GetLibAVCodecName()
    {
-      if (sizeof(void*) == 8)
-         return (wxT("ffmpeg_codecs.") wxT(AV_STRINGIFY(LIBAVCODEC_VERSION_MAJOR)) wxT(".64bit.dylib"));
       return (wxT("libavcodec.") wxT(AV_STRINGIFY(LIBAVCODEC_VERSION_MAJOR)) wxT(".dylib"));
    }
 
    wxString GetLibAVUtilName()
    {
-      if (sizeof(void*) == 8)
-         return (wxT("ffmpeg_utils.") wxT(AV_STRINGIFY(LIBAVUTIL_VERSION_MAJOR)) wxT(".64bit.dylib"));
       return (wxT("libavutil.") wxT(AV_STRINGIFY(LIBAVUTIL_VERSION_MAJOR)) wxT(".dylib"));
    }
 #else
